assets/pages/new-contributions-landing/components/ContributionForm.jsx

```diff
@@ -60,7 +60,7 @@ type PropTypes = {|
   email: string,
   otherAmount: string | null,
   paymentMethod: PaymentMethod,
-  paymentHandlers: { [PaymentMethod]: PaymentHandler | null },
+  thirdPartyPaymentLibraries: PaymentMatrix<PaymentHandler | null>,
```

Comment:
Let's rename `PaymentHandler` type to `ThirdPartyPaymentLibrary`

Reply:
done